db/migrate/20150806184710_create_transactions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTransactions < ActiveRecord::Migration
+  def change
+    create_table :transactions do |t|
+      t.string  :name,           null: false
```

Comment:
the name of whatever transaction they are doing. example -> pay rent: name of company receiving rent $.